unblob/models.py

```diff
@@ -255,30 +255,30 @@ def get_output_dir(self) -> Optional[Path]:
 
 
 class _JSONEncoder(json.JSONEncoder):
-    def default(self, obj):
-        if attrs.has(type(obj)):
```

Comment:
It is OK, but I would just silence the problem here, as I think reading those `o`s has made the code worse.

What about renaming the parameter inside, like

```suggestion
    def default(self, o):
        obj = o   # rename inherited parameter `o`
        if attrs.has(type(obj)):
```

Reply:
I was thinking about this as well. I am okay with renaming it inside the function.

Reply:
worth renaming it in the function to keep linter happy while lowering mental load on our end when reading